ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/verticaltabs/v1/verticaltabs/verticaltabs.html

```diff
@@ -28,8 +28,10 @@
   data-cmp-adaptiveformcontainer-path="${formstructparser.formContainerPath}"
   data-placeholder-text="${wcmmode.edit && 'Please drag Vertical Tabs components here' @ i18n}"
   data-sly-test.widgetId="${'{0}-{1}' @ format=[panel.id, 'widget']}">
-     <div data-sly-call="${label.label @componentId=widgetId, labelValue=panel.label.value, labelVisible=panel.label.visible, bemBlock='cmp-verticaltabs'}" data-sly-unwrap></div>
-     <div data-sly-call="${questionMark.questionMark @componentId=panel.id, longDescription=panel.description, bemBlock='cmp-verticaltabs'}" data-sly-unwrap></div>
+     <div class="cmp-verticaltabs__label-container">
```

Comment:
Can we create a template for label container including other template, this way BEM naming would be consistent.

Reply:
I can create a new template for the label and question mark icon(lableConatiner template), but it will be a bit complicated because we already have templates for both the label and the question mark. We'll need to call these templates from the new one and pass additional arguments.